core/network/warp/sync.cpp

```diff
@@ -119,6 +119,21 @@ namespace kagome::network {
     }
   }
 
+  void WarpSync::unsafe(const BlockHeader &header,
+                        const GrandpaJustification &j,
+                        consensus::grandpa::AuthoritySetId set) {
+    HasAuthoritySetChange{header}.scheduled.value();
+    SL_INFO(log_, "unsafe, block {}, set {}", header.number, set);
```

Comment:
Now it looks like developer's debugging printf. Since it has info log level, please reword it to be more friendly - replace "unsafe" with meaningful description - what this log line should tell.

Was it started, accomplished, in progress or something else?